capa/render/__init__.py

```diff
@@ -26,39 +26,23 @@ def convert_statement_to_result_document(statement):
             "type": "range"
         },
     """
-    if isinstance(statement, capa.engine.And):
-        return {
-            "type": "and",
-        }
-    elif isinstance(statement, capa.engine.Or):
-        return {
-            "type": "or",
-        }
-    elif isinstance(statement, capa.engine.Not):
-        return {
-            "type": "not",
-        }
-    elif isinstance(statement, capa.engine.Some) and statement.count == 0:
-        return {"type": "optional"}
-    elif isinstance(statement, capa.engine.Some) and statement.count > 0:
-        return {
-            "type": "some",
-            "count": statement.count,
-        }
-    elif isinstance(statement, capa.engine.Range):
-        return {
-            "type": "range",
-            "min": statement.min,
-            "max": statement.max,
-            "child": convert_feature_to_result_document(statement.child),
-        }
-    elif isinstance(statement, capa.engine.Subscope):
-        return {
-            "type": "subscope",
-            "subscope": statement.scope,
-        }
-    else:
```

Comment:
nice simplification here, should we keep the `RuntimeError` though?

Reply:
I don't think it is needed. We would have to add an `if` with the rest Statement subclasses (or checking that statement is an `instance` of Statement), but this function is called at https://github.com/Ana06/capa/blob/statement-description/capa/render/__init__.py#L94 after checking that `node` is an instance of Statement. So, I don't see how the error could be trigger. It would be anyway a bug as there is no input the user can provide that should run in an error like this.